harrisonDir/harrisonHomework.txt

```diff
@@ -0,0 +1,113 @@
+PART 1:
```

Comment:
Overall nice work. Take a look at the comments below, and look at the homework solutions for even more specific reasoning. Don't forget to try the value of x as things other than just simple numbers and strings, try booleans, or infinity, or NaN as well!